### PR TITLE
Fix misaligned block in UG

### LINKED ISSUE #2124: Misaligned boxes in User Guide
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

_No response_

### Tell us about your environment

Windows 10

### MarkBind version

V4.0.1

### What did you do? Describe the bug

Boxes in [user guide](https://markbind.org/index.html#simple-syntax-dynamic-content) are not aligned

image

### Steps to reproduce

Go to [user guide](https://markbind.org/index.html#simple-syntax-dynamic-content) and open the panel for "More examples of generating static content"

### Expected behavior

Expected the two boxes to be aligned at the top 

### Actual behavior

Boxes are misaligned at the top 

### Anything else?

nil

### PR DESCRIPTION
**Pull request by yucheng11122017**
**What is the purpose of this pull request?**

- [X] Documentation update
- [X] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Closes #2124
Before: 
image

After: 
image

**Anything you'd like to highlight/discuss:**
nil

**Testing instructions:**
nil

**Proposed commit message: (wrap lines at 72 characters)**
```
Fix misaligned block in UG
```

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [X] Linked all related issues
- [X] No unrelated changes

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ MarkBind extensions to Markdown | `==highlighted text==`<br>`%%grey text%%`<br>`
 An example that uses GFMD syntax for task lists:
 
 <table>
-<tr>
-  <td>
+<tr >
```
Comment (tlylt) [inline docs/index.md:36]: is this empty space intentional? If so could you explain why this is needed ❓

**Review by tlylt — COMMENTED** [review]
Thanks @yucheng11122017 for discovering and fixing this issue, I have never realised it.

Looking at the preview, the second pair of boxes still feels misaligned due to the difference in height, could we do something about it? (perhaps some inline CSS)
image

**Comment by yucheng11122017** [comment]
> Thanks @yucheng11122017 for discovering and fixing this issue, I have never realised it.
> 
> Looking at the preview, the second pair of boxes still feels misaligned due to the difference in height, could we do something about it? (perhaps some inline CSS) image

Hi @tlylt is this better? 
image

**Comment by tlylt** [comment]
> Hi @tlylt is this better? image

Yes much better. Since we are fixing this part, could we perhaps make sure that this part looks nice on mobile as well? Currently, it is unscrollable and part of the content is hidden.
image

**Comment by yucheng11122017** [comment]
Hi @tlylt, two changes 
1. Added scrolling when width of screen is too small 
On mobile: 
> image

2. Added codeWrapPlugIn so that user can wrap the long code if their screen is too small
On mobile: 
> image

This is when there is a bigger screen: 
> image

Hope these are ok!

**Comment by tlylt** [comment]
Hi @yucheng11122017 
This is your bug report:
image
This is the current result:
image
While slightly better, could we perhaps fix it the same way as the second example so that they are better aligned.

Also a side note, I am not sure why the `tip-box` has been changed to `box`, is this a related change required by this PR or due to the deprecation work in another PR?

**Comment by yucheng11122017** [comment]
> This is the current result: image While slightly better, could we perhaps fix it the same way as the second example so that they are better aligned.

Hi @tlylt, sorry about this! I've changed it back to the second example.

File: docs/index.md [hunk 2]
```diff
@@ -46,9 +48,8 @@ An example that uses GFMD syntax for task lists:
 ```
   </td>
   <td>&nbsp;→&nbsp;</td>
-  <td><br>
+  <td>
 <box>
-
```
Comment (tlylt) [inline docs/index.md:51]: This line needs to be retained in order for the `**Things to do:**` to be rendered as markdown, see now:
image
vs before:
image

Reason: https://markbind.org/userGuide/usingHtmlJavaScriptCss.html#markdown-in-html

**Review by tlylt — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (yucheng11122017) [reply docs/index.md:51]: Fixed in 2050515

Comment (tlylt) [reply docs/index.md:51]: Have you taken a look at the preview?
image

Please correct me if I'm seeing things wrong: the first example has a different alignment as compared to the second example. The first one is slightly misaligned and the second one is center-aligned. Are we able to make them align in the same way?

Comment (yucheng11122017) [reply docs/index.md:51]: Ah sorry I thought that was fine based on our previous conversation. 

Computer preview: 
> image

Mobile preview:
> image

**Review by tlylt — APPROVED** [review]

**Pull request merged**